build.cake

```diff
@@ -190,6 +210,139 @@ Task("CopyToLocalPackages")
 Task("Default")
     .IsDependentOn("CopyToLocalPackages");
 
+
+void SignAndPack(string project, string binPath, DotNetCorePackSettings dotNetCorePackSettings){
+    Information("SignAndPack project: " + project);
+    Information("SignAndPack bin path: " + binPath);
+
+    SignAndTimestampBinaries(binPath);
+    
+    DotNetCorePack(project, dotNetCorePackSettings);
+}
+
+private void SignAndTimestampBinaries(string outputDirectory)
+{
+    if (BuildSystem.IsLocalBuild && !signFiles) return;
+
+    Information($"Signing binaries in {outputDirectory}");
+
+    // check that any unsigned libraries, that Octopus Deploy authors, get signed to play nice with security scanning tools
+    // refer: https://octopusdeploy.slack.com/archives/C0K9DNQG5/p1551655877004400
+    // decision re: no signing everything: https://octopusdeploy.slack.com/archives/C0K9DNQG5/p1557938890227100
+    var unsignedExecutablesAndLibraries =
+        GetFiles(outputDirectory + "/{Calamari,Sashimi}*.{exe,dll}")
```

Comment:
Regards to https://github.com/OctopusDeploy/Sashimi.AzureWebApp/pull/27#discussion_r843318023 : the only difference in #27 and #28 should be this line, where it uses a newer file matching, using the current cake tool version.